### DISCUSSION TITLE
buddhism/origination/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#6] [General]
# buddhism/origination/

https://shane0.github.io/buddhism/origination/

**Comment by shane0** [comment]
![o](https://user-images.githubusercontent.com/1454458/270235459-aa79a08a-df78-4116-828b-a75a0f8dbaef.png)

---

The Twelve Links of Dependent Origination (also known as the Twelve Nidanas) is a fundamental teaching in Buddhism. It describes the process by which suffering arises and continues. The Twelve Links can also be seen as a model for the process of evolution.

Here is a brief explanation of the Twelve Links using evolution as the context for each link:

**1. Ignorance:**

Ignorance is the root of all suffering. It is the lack of understanding of the true nature of reality. In the context of evolution, ignorance can be seen as the lack of understanding of the natural world and how it works.

**2. Volitional formations:**

Volitional formations are the mental and physical impulses that arise from ignorance. They are the seeds of karma. In the context of evolution, volitional formations can be seen as the natural instincts and drives that all living things have.

**3. Consciousness:**

Consciousness is the awareness of the present moment. It is the result of volitional formations. In the context of evolution, consciousness can be seen as the ability to perceive and interact with the environment.

**4. Name and form:**

Name and form are the physical and mental characteristics of a being. They are the result of consciousness. In the context of evolution, name and form can be seen as the physical and mental adaptations that have evolved over time in response to the environment.

**5. The six senses:**

The six senses are the organs of sight, hearing, smell, taste, touch, and mind. They are the result of name and form. In the context of evolution, the six senses can be seen as the adaptations that have evolved over time to allow living things to perceive and interact with their environment.

**6. Contact:**

Contact is the meeting of the six senses with their objects. It is the result of the six senses. In the context of evolution, contact can be seen as the interaction between living things and their environment.

**7. Feeling:**

Feeling is the pleasant or unpleasant sensation that arises from contact. It is the result of contact. In the context of evolution, feeling can be seen as the feedback that living things receive from their environment. This feedback helps them to learn and adapt.

**8. Craving:**

Craving is the desire to cling to pleasant feelings and avoid unpleasant feelings. It is the result of feeling. In the context of evolution, craving can be seen as the motivation that drives living things to survive and reproduce.

**9. Grasping:**

Grasping is the act of clinging to pleasant feelings and avoiding unpleasant feelings. It is the result of craving. In the context of evolution, grasping can be seen as the way in which living things interact with their environment in order to get what they need and avoid what they don't need.

**10. Becoming:**

Becoming is the process of creating new karma. It is the result of grasping. In the context of evolution, becoming can be seen as the way in which living things change and adapt over time.

**11. Birth:**

Birth is the beginning of a new life. It is the result of becoming. In the context of evolution, birth can be seen as the emergence of new species.

**12. Old age and death:**

Old age and death are the inevitable end of all life. They are the result of birth. In the context of evolution, old age and death can be seen as the extinction of species.

The Twelve Links of Dependent Origination can be seen as a cycle of suffering. However, it is important to remember that this cycle can be broken. By understanding the Twelve Links, we can learn to let go of ignorance and craving, and to cultivate wisdom and compassion.

It is also important to note that the Twelve Links of Dependent Origination is not a linear process. Each link is interconnected with the others. For example, ignorance can lead to volitional formations, which can lead to consciousness, and so on. However, it is also possible for consciousness to lead to volitional formations, and for volitional formations to lead to ignorance.

The Twelve Links of Dependent Origination is a complex and profound teaching. It is a teaching that can be studied for a lifetime.

**Comment by shane0** [comment]
The twelve links of dependent origination (also known as paticca-samuppada) is a central teaching in Buddhism. It describes the process by which suffering arises and persists. The twelve links are:

1. Ignorance (avijja)
2. Volitional formations (samskara)
3. Consciousness (vijnana)
4. Name-and-form (nama-rupa)
5. The six sense bases (salayatana)
6. Contact (phassa)
7. Feeling (vedana)
8. Craving (tanha)
9. Attachment (upadana)
10. Becoming (bhava)
11. Birth (jati)
12. Old age and death (jaramarana)

The twelve links can be understood in the context of shunyata by realizing that all phenomena are empty of inherent existence. This means that all phenomena, including the twelve links, are impermanent, interconnected, and interdependent. They arise and disappear in dependence on other phenomena, and they have no independent or unchanging reality.

From the perspective of shunyata, the twelve links are not a linear sequence of events. Instead, they are a dynamic web of causes and conditions that give rise to suffering. Ignorance is the root cause of suffering, but it is not the only cause. Volitional formations, consciousness, and the other links also play a role.

However, because all phenomena are empty of inherent existence, there is no such thing as a fixed or permanent self. This means that the suffering that arises from the twelve links is not a personal failing. It is simply the result of our ignorance and our attachment to the illusion of self.

The good news is that the twelve links can also be used to liberate ourselves from suffering. By understanding the nature of shunyata and the interconnectedness of all phenomena, we can begin to let go of our attachments and to cultivate wisdom and compassion.

Here is an example of how the twelve links can be understood in the context of shunyata:

* **Ignorance:** We are ignorant of the true nature of reality, including the emptiness of inherent existence.
* **Volitional formations:** Based on our ignorance, we create volitional formations, such as desires and aversions.
* **Consciousness:** Our consciousness is conditioned by our volitional formations.
* **Name-and-form:** Our name-and-form, which is the physical and mental aggregate that we identify as ourselves, is conditioned by our consciousness.
* **The six sense bases:** The six sense bases, which are the eyes, ears, nose, tongue, body, and mind, are conditioned by our name-and-form.
* **Contact:** Contact between the six sense bases and their objects arises.
* **Feeling:** Feelings arise from contact.
* **Craving:** Craving arises from feelings.
* **Attachment:** We become attached to the objects of our craving.
* **Becoming:** Becoming, or karma, arises from our attachment.
* **Birth:** We are reborn into a new life based on our karma.
* **Old age and death:** Old age and death arise from birth.

This cycle of suffering continues until we break it through the realization of shunyata. When we see that all phenomena are empty of inherent existence, we let go of our attachments and our suffering ceases.

It is important to note that the twelve links are not a linear sequence of events. Instead, they are a dynamic web of causes and conditions that give rise to suffering. The twelve links are also empty of inherent existence, meaning that they are impermanent, interconnected, and interdependent.

The realization of shunyata is a profound awakening that liberates us from the cycle of suffering. By understanding the nature of shunyata and the interconnectedness of all phenomena, we can begin to cultivate wisdom and compassion.

**Comment by shane0** [comment]
https://youtu.be/0iMI_MbxcsY?si=68Pg0ffQB9bIIh7H

**Comment by shane0** [comment]
### Cultivating the Way: Realize No-Self 
with the Wisdom of Dependent Arising

https://www.ctworld.org.tw/Buddhist%20e-Books/CTM/CTM261GM/index.html
